deepface/models/facial_recognition/Yolo.py

```diff
@@ -0,0 +1,44 @@
+# built-in dependencies
+from typing import List
+
+# 3rd party dependencies
+import numpy as np
+
+# project dependencies
+from deepface.models.YoloClientBase import YoloClientBase
+from deepface.models.YoloModel import YoloModel
+from deepface.models.FacialRecognition import FacialRecognition
+from deepface.commons.logger import Logger
+
+logger = Logger()
+
+
+class YoloFacialRecognitionClient(YoloClientBase, FacialRecognition):
+    def __init__(self, model: YoloModel):
+        super().__init__(model)
+        self.model_name = "Yolo"
+        self.input_shape = None
```

Comment:
we want to perform resize_image for all models, but this is not going to be called for yolo. please set something to input shape e.g. (224, 224)

https://github.com/serengil/deepface/blob/master/deepface/modules/preprocessing.py#L77

Reply:
done